eth/backend.go

```diff
@@ -309,41 +309,32 @@ func (s *Ethereum) APIs() []rpc.API {
 		{
 			Namespace: "eth",
 			Version:   "1.0",
-			Service:   NewPublicEthereumAPI(s),
+			Service:   NewEthereumAPI(s),
 			Public:    true,
 		}, {
-			Namespace: "eth",
+			Namespace: "miner",
```

Comment:
This is where `eth_mining` came from. I ended up just moving that function into `EthereumAPI` to free up `MinerAPI` for what was previously the "private` miner API.